src/main/java/kitEditor/Sample.java

```diff
@@ -4,9 +4,9 @@
 import java.util.Random;
 import javax.sound.sampled.*;
 
-class Sample {
+class Sample implements Cloneable {
```

Comment:
Cloneable is not implemented in conventional way.
According to Java docs:
https://docs.oracle.com/javase/8/docs/api/java/lang/Cloneable.html
" By convention, classes that implement this interface should override Object.clone (which is protected) with a public method. See Object.clone() for details on overriding this method. "

I think Cloneable and Object.clone() seem complicated and confusing...
It might be more simple and to the point to add a copy constructor.

private Sample(Sample otherSample) {
    // make myself a deep copy of the otherSample
}

What do you think?

Reply:
```
private Sample(Sample otherSample) {
    // make this a deep copy of the otherSample
}
```
This seems sensible and from what little searching I have done it seems copy constructor is recommended over Object.clone().